File: php/src/Advertisements/Advertisement/Application/Command/PublishAdvertisement/PublishAdvertisementCommand.php
<?php
declare(strict_types=1);

namespace Demo\App\Advertisements\Advertisement\Application\Command\PublishAdvertisement;

final readonly class PublishAdvertisementCommand
{
    public function __construct(
        public string $securityUserId,
        public string $securityUserRole,
        public string $id,
        public string $description,
        public string $email,
        public string $password,
        public string $memberNumber,
        public string $civicCenterId,
    ){}
}

File: kotlin/src/main/kotlin/advertisements/advertisement/application/command/publishAdvertisement/PublishAdvertisementCommand.kt
package advertisements.advertisement.application.command.publishAdvertisement

data class PublishAdvertisementCommand(
    val securityUserId: String,
    val securityUserRole: String,
    val id: String,
    val description: String,
    val email: String,
    val password: String,
    val memberNumber: String,
    val civicCenterId: String
)
